Makefile

```diff
@@ -1,8 +1,37 @@
+SHELL = /bin/bash
+
+prefix ?= /usr/local
+bindir ?= $(prefix)/bin
+libdir ?= $(prefix)/lib
+srcdir = Sources
+
+REPODIR = $(shell pwd)
+BUILDDIR = $(REPODIR)/.build
+SOURCES = $(wildcard $(srcdir)/**/*.swift)
+
+.DEFAULT_GOAL = all
+
 build:
 	swift build -c release
 
 test:
 	swift test
 
-install: build
-	install "$(shell swift build -c release --show-bin-path)/swift-release-notes" /usr/local/bin/
+swift-release-notes: $(SOURCES)
+	@swift build \
+		-c release \
+		--disable-sandbox \
+		--build-path "$(BUILDDIR)"
+
+.PHONY: install
+install: swift-release-notes
+	@install -d "$(bindir)"
+	@install "$(wildcard $(BUILDDIR)/**/release/swift-release-notes)" "$(bindir)"
```

Comment:
Wouldn't it be better to keep using `--show-bin-path` here? Theoretically, someone could run

```
docker run -v $PWD:/host -w /host swift build -c release
```

and end up with incompatible platform binary in `.build`.

Reply:
I use the pattern in all my Swift CLI projects but I haven't verified this with docker. You might be right. I had problems with Makefile installation on my macOS 11 with the current checked-in so when I wanted to add `uninstall` then I did the change for `install`.

Reply:
FYI, this would an issue (and I'm about to add Linux support :) )

```
❯ docker run --rm -v $PWD:/host -w /host swiftarm/swift:5.5.2-ubuntu-focal swift build -c release
[1/1] Planning build
remark: Incremental compilation has been disabled: it is not compatible with whole module optimization[1/2] Compiling swift_release_notes main.swift
[3/3] Linking swift-release-notes
[3/3] Build complete!
~/P/S/swift-release-notes on  executableProduct took 3s 
❯ make install
[3/3] Build complete!
install /Users/sas/Projects/SPI/swift-release-notes/.build/aarch64-unknown-linux-gnu/release/swift-release-notes /Users/sas/Projects/SPI/swift-release-notes/.build/arm64-apple-macosx/release/swift-release-notes /usr/local/bin
```

(I've replace `@install` with `@echo install`)